# The joint angles for a single leg
string leg_name
float64 coxa_angle
float64 femur_angle
float64 tibia_angle
